composer.json

```diff
@@ -23,9 +23,7 @@
   "require"    : {
     "php": ">=5.4.0",
     "consolidation/robo": "~1",
-    "joomla/github": "~1.3",
-    "greencape/coding-standards": "~1",
```

Comment:
Then it's because it's lacking the updates to the `joomla-testing-robo` repo
Since I'm taking longer than I though with the whole container system testing I can push a PR with only that specific part (phpcs) to both repos